biometrics_genotype_0.2.4/README.md

```diff
@@ -0,0 +1,51 @@
+# CWL and Dockerfile for running biometrics genotype tool.
+
+| Tool | Version | Location |
+|--- |--- |--- |
+| biometrics   | 0.2.4   |  <https://pypi.org/project/biometrics/> |
+
```

Comment:
Can you point to the docker file in your github repo

Reply:
i updated it. this is what you meant?

Reply:
Nah we can just have it pointing to the tool on github, and mention that the Docker file is from that repo.